python/requirements.txt

```diff
@@ -41,7 +41,7 @@ pyyaml
 rich
 gpustat>=1.0.0
 opentelemetry-sdk
-fastapi
+fastapi==0.99.1
```

Comment:
We should avoid pinning to specific versions wherever possible because it causes people a lot of headaches. Is there any reason we need exactly this version, or can it be `<= 0.99.1`?

Reply:
It can be `<= 0.99.1`.